backend/core/admin.py

```diff
@@ -37,3 +42,13 @@ class EntityDescriptionAdmin(admin.ModelAdmin):
         contributions_fieldset,
         description_source_fieldset,
     ]
+
+
+def get_queryset_matching_parent_source(
+    admin: admin.StackedInline | admin.TabularInline,
+    db_field: RelatedField,
+    request: HttpRequest,
+) -> QuerySet:
+    parent_id = re.search(f"\d+", request.path).group(0)
```

Comment:
I'm getting an AttributeError when click `Add Agent Description` in Django Admin, because `re.search()` can't find a match.

Suggestion:
```python
    re_result = re.search(r"\d+", request.path)
    if re_result is None:
        return db_field.remote_field.model.objects.none() # or .all() depending on what you want the default to be.
    parent_id = re_result.group(0)
```

Also note that instead of an `f`-string, you probably want to use an `r` string in your regex. (Pylance is warning me about this: `Unsupported escape sequence in string literal`).